packages/@angular/cli/blueprints/ng/files/package.json

```diff
@@ -12,11 +12,13 @@
   },
   "private": true,
   "dependencies": {
+    "@angular/animations": "^4.0.0",
```

Comment:
Could we add this dependency on https://github.com/angular/angular-cli/pull/5786 ? Or update this PR to integrate `BrowserAnimationsModule`? simply adding the dependency to the package wouldn't make it any easier to use animations or angular material as requested on https://github.com/angular/angular-cli/issues/5928. It would also take up space from apps that do not need animations.